### DISCUSSION TITLE
Hide controller

### DISCUSSION THREAD

**Discussion by Rini0404** [ihmpavel/expo-video-player#729] [Q&A] [answered]
Hello! 

Is there any way to hide the pause/play button? For example, it auto-hides in the beginning, but after it loops it shows it again.

Thank you!

**Comment by ihmpavel** [comment, marked as answer]
Maybe https://github.com/ihmpavel/expo-video-player/blob/master/example-app/App.tsx#L97-L100 should help?

After the end you can hide/change the icons. 

Or `autoHidePlayer` prop.

**Reply by Rini0404** [reply]
Thank you!
